### ISSUE TITLE
Setting the SRS prop for WMSTileLayer is not respected

### ISSUE THREAD

**Issue by mdfischer** [PaulLeCam/react-leaflet#245]
### Expected behavior

The srs prop, when passed to a WMSTileLayer, should be reflected in the WMS query string that gets sent out to the tile server.
### Actual behavior

The srs query string parameter is sent out as `&srs=EPSG%3A3857` on the initial load, regardless of what is provided for the srs prop in WMSTileLayer, e.g. `<WMSTileLayer srs='EPSG:4326' etc.>`.  Subsequent loads/reloads do result in the provided srs prop ('EPSG:4326') coming through in the request.
### Sample implementation

Using a custom React component, `this.getWMSParams()` grabs the set of props that are set by `getDefaultProps()`, which includes `srs: 'EPSG:4326'`:

```
var RasterOverlay = React.createClass({
...
render: function () {
    var params = this.getWMSParams();
    console.log('RasterOverlay::render params:', params); // confirms correct loading of default props
    return (
      <div className={styles.overlay}>
        <WMSTileLayer
          url={NCWMS_URL}
          format={params.format}
          transparent={params.transparent}
          styles={params.styles}
          version={params.version}
          srs={params.srs}
          logscale={params.logscale}
          origin={params.origin}
          layers={params.layers}
          time={params.time}      
        />
      </div>
    );
  },
});




```

**Comment by PaulLeCam** [comment]
What version of the library are you using please?
Dynamic props for WMSTileLayer are only supported in v1.0.0-rc.2.

**Comment by mdfischer** [comment]
I am using the updated v1.0.0-rc.2 (your 'next' branch).  I have confirmed that the other parameters update appropriately and a new WMS request is sent out (as you solved in issue #237), but then found this new issue with the SRS prop.

**Comment by mdfischer** [comment]
Actually, this is my mistake.  I didn't realise that the CRS property needs to be set by the parent Map, which the enclosed WMSTileLayer then inherits (successfully).  So, I was seeing the default EPSG:3857 being passed down from the Map on the initial load, because I had no CRS/SRS set.  Sorry for the confusion.
